src/main/java/org/mvplugins/multiverse/core/api/SafeTTeleporter.java

```diff
@@ -14,6 +14,7 @@
 /**
  * Used to safely teleport people.
  */
+@Deprecated
 @Contract
 public interface SafeTTeleporter extends Teleporter {
```

Comment:
Why leave it in at all?

Reply:
Bcu of the world manager, now that it’s nuked, it can be removed